superset-frontend/src/views/CRUD/data/database/DatabaseModal/index.test.jsx

```diff
@@ -18,13 +18,24 @@
  */
 import React from 'react';
 import fetchMock from 'fetch-mock';
-// import userEvent from '@testing-library/user-event';
-import { render, screen } from 'spec/helpers/testing-library';
+import userEvent from '@testing-library/user-event';
+import {
+  render,
+  screen,
+  within,
+  cleanup,
+  act,
+} from 'spec/helpers/testing-library';
+/* -- These imports are used for the mock functions that currently don't work
+import {
+  testDatabaseConnection,
+  useSingleViewResource,
+} from 'src/views/CRUD/hooks'; */
 import DatabaseModal from './index';
 
 const dbProps = {
   show: true,
-  databaseId: 10,
+  // databaseId: 10,
```

Comment:
just remove this since you are commenting it out

Reply:
Oops! Missed that one in the cleanup. Fixed in [`this commit`](https://github.com/apache/superset/pull/15394/commits/bd15afe23141700eed0b34b97681c305c08b66e0).